packages/db/src/schema/list.coffee.md

```diff
@@ -63,4 +63,4 @@ schemas.map( ({name, owner}) => {
       
 ## Dependencies
 
-    utils = require '@nikitajs/engine/lib/utils'
+    utils = require '@nikitajs/engine/src/utils'
```

Comment:
certainly not

Reply:
> make sure you checked all utils (including lxd)

I did

Reply:
yes, it must now be utils

Reply:
> certainly not

Do you mean I should apply `utils = require './utils'` even if there is no utils in `db`?

Reply:
No, I meant to require "lib" and not "src".